src/clients/dotnet/dotnet_bindings.zig

```diff
@@ -515,23 +512,5 @@ pub fn main() !void {
     var buffer = std.ArrayList(u8).init(allocator);
     try generate_bindings(&buffer);
 
-    try std.fs.cwd().writeFile(.{ .sub_path = output_file, .data = buffer.items });
-}
-
-const testing = std.testing;
-
-test "bindings dotnet" {
```

Comment:
Should we no run the bindings through `zig build test`? Makes sense given the CI already calls `zig build clients:X` which runs them.

IIRC, them being imported to `unit_tests.zig` was the issue the `vsr` import couldn't work. So it's nice that it does now.

Reply:
Yeah, that's the major pivot: instead of veryfing freshness through _tests_, we now special-case CI, and just fail the build there if the files are not fresh. The relevant part is here: 

https://github.com/tigerbeetle/tigerbeetle/pull/2193/files#diff-f87bb3596894756629bc39d595fb18d479dc4edf168d93a911cadcb060f10fccR985

And yeah, that's exactly what allows us to get rid of the tests, add a proper vsr module dependency, and move the files back into `client/$lang` folders!